### DISCUSSION TITLE
Issue with Multi-Language while using Many-To-Many Object Relation with Tag View

### DISCUSSION THREAD

**Discussion by rushitkhunt1996** [pimcore/pimcore#17759] [Q&A / Support]
Hi,

on Pimcore 11, We are facing one issue.
We used  Many-To-Many Object Relation field and we selected display mode as a Tag Field.
![image](https://github.com/user-attachments/assets/8c62a407-cbea-4971-a238-184ba2753128)
After this configuration, On product object we can see the relational field is converted on select(drop-down) and that value will be loaded from the Color Master object and in the Color master we have a Localized field for color name.

With the Tag Field display mode, my Product color relational field looks like the below screenshot and I have selected "Red" as a color.
The screenshot is from English language user.
![image](https://github.com/user-attachments/assets/5012b86e-5667-4c43-9170-06b7434d6e37)

Now, we log in with a German language user and I am expecting the selected value should be visible in German same like relations field, but it is visible in English only.
The Screenshot is from German language user.
![image](https://github.com/user-attachments/assets/0ac65504-9db2-4673-8a05-d683d2914ad4)

Now when I click on the drop-down, the selected value is in German in the options list, but while opening the product it's in English.
After clicking on the drop-down with German Language User.
![image](https://github.com/user-attachments/assets/3b868218-f409-42f0-a9ed-ebf47a2f9b49)

**Comment by ghost** [comment]
Hi,
usually the displayed locale in relation fields is taken from the configured default locale:
```
framework:
    default_locale: 'de'
```
If none is set, EN will be displayed.
I don't think, this is connected to the users profile locale.
But you're right, it is contradictory that DE is shown in the options list  ...
